{"uid":"4b4757e66a1912dae1a509f688f20b0f","name":"categories","children":[{"name":"Test defects","children":[{"name":"requests.exceptions.ConnectionError: HTTPConnectionPool(host='192.168.0.105', port=3000): Max retries exceeded with url: /api/user/login (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000235661CFE50>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))","children":[{"name":"test_login[result0]","uid":"855ec7744f974935","parentUid":"cd80505f89dd893665d19b861c6dd328","status":"broken","time":{"start":1706024563159,"stop":1706024565173,"duration":2014},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":2,"retriesStatusChange":true,"parameters":["('001', 'succ-login', 'user', '192.168.0.105:3000', '/api/user/login', 'post', '0', '{\"username\":\"test3\",\"password\":\"test3\"}', '{\"errno\":0,\"data\":null,\"message\":\"登录成功\"}', '{\\n  \"errno\": 0,\\n  \"message\": \"登录成功\"\\n}', 'pass', 'miss')"],"tags":[]}],"uid":"cd80505f89dd893665d19b861c6dd328"},{"name":"requests.exceptions.ConnectionError: HTTPConnectionPool(host='192.168.0.105', port=3000): Max retries exceeded with url: /api/user/login (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002356580D570>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))","children":[{"name":"test_login[result1]","uid":"37e0df3b92f127b0","parentUid":"7c23fee221db0d9448fe1a481f822c87","status":"broken","time":{"start":1706024565277,"stop":1706024567281,"duration":2004},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":2,"retriesStatusChange":true,"parameters":["('002', 'error-username_fail', 'user', '192.168.0.105:3000', '/api/user/login', 'post', '0', '{\"username\":\"test\",\"password\":\"test3\"}', '{\"errno\":1002,\"message\":\"登录失败\"}', '{\\n    \"errno\": 1002,\\n    \"message\": \"登录失败\"\\n}', 'pass', 'miss')"],"tags":[]}],"uid":"7c23fee221db0d9448fe1a481f822c87"},{"name":"requests.exceptions.ConnectionError: HTTPConnectionPool(host='192.168.0.105', port=3000): Max retries exceeded with url: /api/user/login (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000023566177280>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))","children":[{"name":"test_login[result2]","uid":"4d5227dfd64c8c2f","parentUid":"a6db092a4d75452b6d934de66ed65501","status":"broken","time":{"start":1706024567386,"stop":1706024569404,"duration":2018},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":2,"retriesStatusChange":true,"parameters":["('003', 'error-password_fail', 'user', '192.168.0.105:3000', '/api/user/login', 'post', '0', '{\"username\":\"test3\",\"password\":\"test\"}', '{\"errno\":1002,\"message\":\"登录失败\"}', '{\\n    \"errno\": 1002,\\n    \"message\": \"登录失败\"\\n}', 'pass', 'miss')"],"tags":[]}],"uid":"a6db092a4d75452b6d934de66ed65501"},{"name":"TypeError: list indices must be integers or slices, not str","children":[{"name":"test_login","uid":"ac58a6a18af5deec","parentUid":"a6ca840760d68beef4d0b02ac437317c","status":"broken","time":{"start":1706120658304,"stop":1706120658304,"duration":0},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":5,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"a6ca840760d68beef4d0b02ac437317c"}],"uid":"bdbf199525818fae7a8651db9eafe741"},{"name":"Product defects","children":[{"name":"AssertionError: assert 0 == 1001","children":[{"name":"test_register[result0]","uid":"7cab27e631aff5d2","parentUid":"c81bdc355b5c70fced495d9da7b8200b","status":"failed","time":{"start":1706174785104,"stop":1706174785123,"duration":19},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":3,"retriesStatusChange":true,"parameters":["('004', 'succ-register', 'user', '192.168.0.105:3000', '/api/user/register', 'post', '0', '{\"username\":\"test222333\",\"password\":\"test222333\",\"confirm_password\": \"test222333\"}', '{\"errno\":0,\"data\":{},\"message\":\"注册成功\"}', '{\\n    \"errno\": 0,\\n    \"data\": {\\n        \"username\": \"test222333\",\\n        \"_id\": \"6599251dba8c71a12392c891\",\\n        \"createdAt\": \"2024-01-06T10:02:05.636Z\",\\n        \"updatedAt\": \"2024-01-06T10:02:05.636Z\",\\n        \"__v\": 0\\n    },\\n    \"message\": \"注册成功\"\\n}', 'miss', 'miss')"],"tags":[]}],"uid":"c81bdc355b5c70fced495d9da7b8200b"},{"name":"AssertionError: assert 30001 == 0","children":[{"name":"test_order_submit[result1]","uid":"6328afffd0309615","parentUid":"a422bffc4768a714eed56a6eb56ffcf2","status":"failed","time":{"start":1706176287772,"stop":1706176287780,"duration":8},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":4,"retriesStatusChange":true,"parameters":["('002', 'fail_order_commit', 'order', '192.168.0.105:3000', '/api/order', 'post', '1', 'null', '{\"errno\":30001, \"message\":\"创建订单出错\"}', '{\"errno\":30001, \"message\":\"创建订单出错\"}', 'pass', 'ignore')"],"tags":[]}],"uid":"a422bffc4768a714eed56a6eb56ffcf2"}],"uid":"8fb3a91ba5aaf9de24cc8a92edc82b5d"}]}